lib/ript/dsl/primitives/common.rb

```diff
@@ -18,10 +18,16 @@ def ports(*args)
               if port.class == Range
                 @ports << "#{port.begin}:#{port.end}"
               else
+                if not port.is_a? Numeric
+                  raise PortError, "Port #{port} is not numeric cannot continue"
+                end
```

Comment:
Will change was following how the code was previously written as per https://github.com/bulletproofnetworks/ript/blob/e3e59b3be26e2f342f05bb4dd36ef9b0d048832b/lib/ript/dsl/primitives/common.rb#L72-L74